src/app/static/src/app/components/forms/types.ts

```diff
@@ -1,20 +1,26 @@
 import {Dict} from "../../types";
 
-export interface DynamicControlSection {
+export type DynamicControlSection = {
```

Comment:
There's practically no difference between interfaces and types in typescript these days. Making these types felt a bit tidier.